md/wildfly-bundle.md

```diff
@@ -164,19 +170,38 @@ WildFly can be shut down by executing the following script:
 
 You can also press Ctrl + C.
 
+<a id="post-install" />
```

Comment:
I haven't added it. 
If you are sure it's not used, I can remove it.
Can you test?
Let me know.